app/models/patient.rb

```diff
@@ -129,6 +129,136 @@ def latest_transfer
+      .should_be_monitoring
+      .has_usable_preferred_contact_method
+      .where('patients.id = patients.responder_id')
+      .where(pause_notifications: false)
+      .where('patients.last_assessment_reminder_sent <= ? OR patients.last_assessment_reminder_sent IS NULL', 12.hours.ago)
+      .or(hoh_eligible_for_any_dependents)
+  }
+
+  # Non-eligible contact methods are: ['Unknown', 'Opt-out', '', nil]
+  scope :has_usable_preferred_contact_method, lambda {
+    where.not(preferred_contact_method: ['Unknown', 'Opt-out', '', nil])
+  }
+
+  # Check if the HoH has any dependents that make them eligible to
+  # recieve notifications on the dependent's behalf.
+  # The joined table is referred to as `dependents_patients` and is used for all
+  # checks made on the dependents. Anywhere the `patients` table is specified or
+  # where a hash is used for a condition are checks made on the HoH.
+  #
+  # A dependent patient makes the HoH reminder eligible if:
+  # - HoH is not purged
+  # - HoH has a usable preferred contact method
+  # - HoH has not paused notifications
+  # - current HoH local time is within the preferred contact time window
+  # - patient has monitoring ON
+  # - patient is not purged
+  # - patient is in isolation
+  #   OR
+  #   patient is in continuous exposure
+  #   OR
+  #   patient last date of exposure is on or after (today - monitoring_period_days)
+  #   OR
+  #   patient last date of exposure is null and created at of exposure is on or after (today - monitoring_period_days)
+  scope :hoh_eligible_for_any_dependents, lambda {
+    joins(:dependents)
+      .where(purged: false)
+      .where(head_of_household: true)
+      .where('patients.id = patients.responder_id')
+      .where(
+        # Ignore any joined rows where the dependents_patients is the HoH itself.
+        'dependents_patients.id != dependents_patients.responder_id'
+      )
+      .has_usable_preferred_contact_method
+      .where(
+        # HoH is unconditionally ineligible if it has paused notifications
+        pause_notifications: false
+      )
+      .where('dependents_patients.monitoring = ?', true)
+      .where('dependents_patients.purged = ?', false)
+      .where(
+        'dependents_patients.isolation = ? '\
+        'OR dependents_patients.continuous_exposure = ? '\
+        'OR dependents_patients.last_date_of_exposure >= DATE_SUB(DATE(CONVERT_TZ(?, "UTC", dependents_patients.time_zone)), INTERVAL ? DAY) '\
+        'OR ('\
+        '  dependents_patients.last_date_of_exposure IS NULL '\
+        '  AND dependents_patients.created_at >= DATE_SUB(DATE(CONVERT_TZ(?, "UTC", dependents_patients.time_zone)), INTERVAL ? DAY)'\
+        ')',
+        true,
+        true,
+        Time.now.getlocal('-00:00'),
+        ADMIN_OPTIONS['monitoring_period_days'],
+        Time.now.getlocal('-00:00'),
+        ADMIN_OPTIONS['monitoring_period_days']
+      )
+      .within_preferred_contact_time
```

Comment:
Shouldn't we still be checking the following on the HoH in these cases?

- `has_not_reported_recently_enough_for_reminder_exlusion` (I could see us wanting to change the logic to not do this in these cases, but the current logic does check this).
- `last_assessment_reminder_sent`

Reply:
In our previous conversations for the logic of `has_not_reported_recently_enough_for_reminder_exlusion` for dependents, I believe we concluded that we wanted to keep the current way the logic works - which does not take into account the last time dependents have reported.

Reply:
Would we really want a HoH filling out their own report by not for a dependent to result in excluding them from getting a reminder? I thought the intended functionality was to send a reminder if any dependent still needs a report even if the HoH has submitted their own already.

Reply:
Yes, that assumption is correct.

Reply:
> Would we really want a HoH filling out their own report by not for a dependent to result in excluding them from getting a reminder?

Agreed, we wouldn't. Meaning NOT including the `has_not_reported_recently` is what we want. So I do want to point out that this does appear to be a bug in our current eligibility scope then, as we do check this in `reminder_eligible` with `.where('latest_assessment_at < ? OR latest_assessment_at IS NULL', Time.now.in_time_zone('Eastern Time (US & Canada)').beginning_of_day)`.

I do also still think we'd want to check `last_assessment_reminder_sent` here though.

Reply:
I'm going to pull out the scope logic for `last_assessment_reminder_sent` so we can avoid duplicating that code

Reply:
@holmesie @kylietmo This now uses the scope `reminder_not_sent_recently` in `better_reminder_eligible` and `has_eligible_dependents`